doc/examples/effects/combined_effects/lib/main.dart

```diff
@@ -35,13 +35,19 @@ class MyGame extends BaseGame with TapDetector {
     final dx = details.localPosition.dx;
     final dy = details.localPosition.dy;
     greenSquare.clearEffects();
+    final Vector2 currentTap = Vector2(dx, dy);
```

Comment:
nit: `details.localPosition.toVector2()` :p 
I am addicted haha

Reply:
Ah nice, forgot about these old test when I added the vector extension.
I'll add them now. :)